sd_webui_bayesian_merger/merger.py

```diff
@@ -225,6 +226,40 @@ def merge_block(self, current_bases: Dict, thetas: Dict, key: str) -> Dict:
         beta = current_bases["beta"]
         if self.cfg.merge_mode == "sum_twice":
             return (1 - beta) * ((1 - alpha) * t0 + alpha * t1) + beta * t2
+        elif self.cfg.merge_mode == "tensor_sum":
+            dim = t0.dim()
+            if alpha+beta <= 1 :
+                tt=t0.clone()
```

Comment:
I am just trying to avoid directly modifying input variables since I don't want to modify the variable in the function that called the merge_blocks function. It's about avoiding modifying variables that are passed by reference so that you don't change their value in the function that passed the variable which can lead to undefined behavior. (This was how it was done in supermerger, and I personally agree with it as I come from a Java/C/C++/JavaScript background where this is an issue, and I am not sure how python handles this, so better to be safe than sorry.)